test/url-import.html

```diff
@@ -0,0 +1,13 @@
+<script>
+  // override edge's URL Implementation that does not convert spaces to %20
+  // must be run before Polymer's url polyfill
```

Comment:
we try to do a thing in it where we pass it Polymer.Base.async which is why I call it our polyfill